docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,24 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Air-gapped registries
```

Comment:
@clnperez: Good question. "Registry on an air-gapped network" is probably a better description than "air-gapped registry." Does that add clarity?